src/OpenSSL/SSL.py

```diff
@@ -272,7 +272,28 @@
 SSL_CB_HANDSHAKE_DONE: int = _lib.SSL_CB_HANDSHAKE_DONE
 
 _T = TypeVar("_T")
-_SetVerifyCallback = Callable[["Connection", X509, int, int, int], bool]
+
+
+class _NoOverlappingProtocols:
+    pass
+
+
+NO_OVERLAPPING_PROTOCOLS = _NoOverlappingProtocols()
```

Comment:
Not 100% this is the right way to do it, guidance would be appreciated.

Reply:
https://peps.python.org/pep-0661/ seems to be a bit in limbo. This seems like a good enough workaround to me for now.

Reply:
Thanks for the pointer, I had no idea a PEP was submitted for this.